xonsh/xonfig.py

```diff
@@ -15,6 +15,8 @@
 import contextlib
 import collections
 
+from tempfile import TemporaryDirectory
```

Comment:
This should just be `import tempfile`, like the imports above, so that we get the amalgamate benefit